### ISSUE TITLE
Compat: CS2Fixes compatibility issues

### ISSUE THREAD

**Issue by roflmuffin** [roflmuffin/CounterStrikeSharp#94]
We need to handle compatibility with CS2Fixes, at least in a basic sense, that means:

- Don't send "player_chat" event if CS2Fixes is found as they send this also.

**Comment by KillStr3aK** [comment]
if I recall correctly we can query what metamod plugins are running so it should be an easy thing

**Comment by roflmuffin** [comment]
Yeah, we should also return an accurate build number in our own MM version so if anyone else queries us they get that too

**Comment by mihaigsm2003** [comment]
with latest vers V54 and cs2fixes if change map server crash, also i see no problems with CSS V50

**Comment by KillStr3aK** [comment]
> with latest vers V54 and cs2fixes if change map server crash, also i see no problems with CSS V50

do you have cs2fixes installed?

**Comment by mihaigsm2003** [comment]
> > with latest vers V54 and cs2fixes if change map server crash, also i see no problems with CSS V50
> 
> do you have cs2fixes installed?

yes

**Comment by asdfxD** [comment]
still comp issues. if using bot latest cs2fixes and cssharp, no commands from cs2fixes are working, if u say !kick for exampel it says "target not found" and other chat triggers aren't working too....

**Comment by gjonezin** [comment]
I'm interested in adding CSSharp to a server with just metamod and CS2fixes. My goal is to use the Warcraft mod, listed as an example, alongside the ZE mod. Has anyone done this successfully? 

Is it still only commands that are not compatible? Also, what order did you load successfully? I have a private server already setup with metamod and CS2fixes.

**Comment by KillStr3aK** [comment]
warcraft is not 

> I'm interested in adding CSSharp to a server with just metamod and CS2fixes. My goal is to use the Warcraft mod, listed as an example, alongside the ZE mod. Has anyone done this successfully?
> 
> Is it still only commands that are not compatible? Also, what order did you load successfully? I have a private server already setup with metamod and CS2fixes.

Warcraft is not meant to be installed as it contains legacy code from the previous project
